tests/integ/modin/series/test_apply.py

```diff
@@ -171,9 +189,28 @@ def test_apply_and_map_basic_with_type_hints(data, func, return_type):
     )
 
 
-@pytest.mark.parametrize("data,func,return_type", BASIC_DATA_FUNC_RETURN_TYPE_MAP)
+@pytest.mark.parametrize(
+    "data",
+    [
+        [1, 2, 3, None],
+        [1, 2, 3, None],
+        ["s", "n", "o", "w"],
+        [1.0, 1.5, 2.0],
+        [True, True, False],
+        [bytes("snow", "utf-8"), bytes("flake", "utf-8")],
+        [1, 2, 3],
+        [1, 2, 3],
+        [None, 1],
+        [np.nan, None, 2.1],
+        param(
+            [native_pd.Timedelta(1), native_pd.Timedelta(2)],
+            id="input_timedleta_columns",
+            marks=pytest.mark.xfail(strict=True, raises=NotImplementedError),
+        ),
+    ],
+)
 @sql_count_checker(query_count=4, udf_count=1)
```

Comment:
This test function used to be parametrized by `BASIC_DATA_FUNC_RETURN_TYPE_MAP`, but the test function confusingly used the `data` from that constant but ignored the `func` and `return_type` params. I wanted to attach `xfail`s to the `BASIC_DATA_FUNC_RETURN_TYPE_MAP` params that reflect what happens when you call the `func` with the given input, but this function applies a different `func`, so I'm copying just the test data into a new parameter list here, and using custom `xfail`s.